pkg/frontend/kubeactions/kubeactions.go

```diff
@@ -179,13 +178,31 @@ func (ka *kubeactions) CreateOrUpdate(ctx context.Context, oc *api.OpenShiftClus
 		return err
 	}
 
+	objbytes, err := obj.MarshalJSON()
+	if err != nil {
+		return err
+	}
+
 	_, err = dyn.Resource(*gvr).Namespace(namespace).Update(obj, metav1.UpdateOptions{})
 	if !errors.IsNotFound(err) {
 		return err
 	}
 
+	if errors.IsNotFound(err) {
+		createaction = true
+	}
+
 	_, err = dyn.Resource(*gvr).Namespace(namespace).Create(obj, metav1.CreateOptions{})
-	return err
+	if err != nil {
+		return err
+	}
+
+	if createaction {
+		ka.log.Infof("Creating obj: %s", objbytes)
+	} else {
+		ka.log.Infof("Updating obj: %s", objbytes)
+	}
+	return nil
 }
 
 func (ka *kubeactions) Delete(ctx context.Context, oc *api.OpenShiftCluster, groupKind, namespace, name string) error {
```

Comment:
I'm not sure what to do about logging Deletes. 
The Delete with GVK and name will appear in the cluster's audit log.
Doing a Get before deleting just for the purpose of logging seems to be against the idea to keep geneva actions simple and less prone to API flapping.